mappings/net/minecraft/recipe/RecipeManager.mapping

```diff
@@ -1,15 +1,29 @@
 CLASS net/minecraft/class_1863 net/minecraft/recipe/RecipeManager
 	FIELD field_19359 GSON Lcom/google/gson/Gson;
-	FIELD field_9023 recipeMap Ljava/util/Map;
-	FIELD field_9024 hadErrors Z
+	FIELD field_9023 recipes Ljava/util/Map;
+	FIELD field_9024 errored Z
 	FIELD field_9027 LOGGER Lorg/apache/logging/log4j/Logger;
-	METHOD method_17717 getAllForType (Lnet/minecraft/class_3956;)Ljava/util/Map;
+	METHOD apply (Ljava/util/Map;Lnet/minecraft/class_3300;Lnet/minecraft/class_3695;)V
+		ARG 1 preparedObject
+		ARG 2 resourceManager
+		ARG 3 profiler
+	METHOD method_17717 getAllOfType (Lnet/minecraft/class_3956;)Ljava/util/Map;
```

Comment:
```suggestion
	METHOD method_17717 getAllOf (Lnet/minecraft/class_3956;)Ljava/util/Map;
```
The parameter is already a recipe type.

Reply:
Yes, but we say "get all the recipes of type X", not just "get all the recipes of X".